jenkins_jobs.yml

```diff
@@ -30,6 +32,17 @@
         value: false
       when: http_sslVerify.config_value != 'false'
 
+    - name: Locally clone Jenkins job configs
```

Comment:
Moved this around while I was debugging an issue with weird directories getting created (`'['/'/['` or something like that....).

After two hours of wasting time looking and testing, I realized that the issue was I had recently switched to using a list on the `jenkins_job_config_file_src` variable.

I have fixed this bug, and now the list no longer overloads the original variable. I ended up adding a new destination variable and fixing the variable overload problem.